### DISCUSSION TITLE
Can messages be generated for dependencies that are years and years out of date?

### DISCUSSION THREAD

**Discussion by aSemy** [renovatebot/renovate#9293] [Request Help]
I'd like to modernise a large, old project that has a lot of dependencies. I think a dependency bot like Renovate could help - but I have one specific question (see title).

For example, there is a dependency declared on [com.jayway.awaitility:awaitility:1.7.0](https://mvnrepository.com/artifact/com.jayway.awaitility/awaitility). This dependency was released in 2015.

From what I understand, Renovate would not see any problem with this dependency, as it is on the latest version. However, really it's out of date. The dependency was moved to [org.awaitility:awaitility](https://mvnrepository.com/artifact/org.awaitility/awaitility), which released version 4 last year.

Other dependencies are in a similar situation (and not just in Maven). They have been abandoned, or replaced. 

Ignoring the task of *how* to update these old dependencies, can Renovate provide these warnings, so I can make a start?

**Comment by rarkins** [comment]
You would probably be interested to follow https://github.com/renovatebot/renovate/issues/5535 for when it's implemented

**Reply by aSemy** [reply]
Ah, thanks. That would help upgrading known moved dependencies, like the Awaitility? But #5535 isn't specifically about identifying dependencies that might be outdated?

**Reply by rarkins** [reply]
There's possibly another issue - from even longer ago - about identifying and replacing outdated dependencies. It's definitely something we'd like to do, with some form of community-sourced list.

**Comment by HonkingGoose** [comment]
I suspect we have some issues open for this feature where Renovate helps with deprecated/dead dependencies. The issues listed below _might_ be duplicates, or they might not be. At the very least they seem to cover similar ground.

- https://github.com/renovatebot/renovate/issues/2084
- https://github.com/renovatebot/renovate/issues/3339
- https://github.com/renovatebot/renovate/issues/2223
- https://github.com/renovatebot/renovate/issues/2290
- https://github.com/renovatebot/renovate/issues/2305

I'll let @rarkins decide on what to keep and mark as duplicate of... 😉

**Reply by aSemy** [reply]
Thanks, I think #2305 is the closest thing to what I'm asking: identifying any dependencies that haven't had a release in years.
